rosdep/base.yaml

```
@@ -574,11 +575,25 @@ clang-tidy:
   fedora: [clang-tools-extra]
   gentoo: [sys-devel/clang]
   nixos: [clang]
+  opensuse: [clang]
   rhel:
     '*': [clang-tools-extra]
     '7': null
   ubuntu:
     '*': [clang-tidy]
+clangd:
+  alpine: [clang-extra-tools]
+  arch: [clang]
+  debian: [clangd]
+  fedora: [clang-tools-extra]
+  gentoo: [sys-devel/clang]
+  nixos: [clang]
+  opensuse: [clang]
+  rhel:
+    '*': [clang-tools-extra]
+    '7': null
```

We don't need to worry about RHEL-7 anymore:

```suggestion
  rhel: [clang-tools-extra]
```